ts/IterableInboxMessageDisplay.tsx

```diff
@@ -74,13 +74,14 @@ const IterableInboxMessageDisplay = ({
    const handleHTMLMessage = (event: any) => {
       let url = event.nativeEvent.data
 
-      Iterable.trackInAppClick(rowViewModel.inAppMessage, IterableInAppLocation.inApp, url)
+      Iterable.trackInAppClick(rowViewModel.inAppMessage, IterableInAppLocation.inbox, url)
       
       if(url === 'iterable://delete') {
          deleteRow(rowViewModel.inAppMessage.messageId)
          returnToInbox()
       } else if(url === 'iterable://dismiss') {
          returnToInbox()
+         Iterable.trackInAppClose(rowViewModel.inAppMessage, IterableInAppLocation.inbox, IterableInAppCloseSource.link, 'iterable://dismiss')
```

Comment:
Whoops. I think our `trackInAppClose` isn't complete, the `clickedUrl` field here should just be `null`, so we should build that support in.

Reply:
I will make the update.

Reply:
I added undefined for the clickedUrl field